packages/paste-core/components/editable-code-block/src/EditableCodeBlockWrapper.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {Box} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+import {styled, css} from '@twilio-paste/styling-library';
+
+const StyledEditableCodeBlockWrapper = styled(Box)(
+  css({
+    backgroundColor: 'rgb(1, 22, 39)', // Matches the Night Owl background theme
+    borderRadius: 'borderRadius20',
+    overflow: 'hidden',
+  })
```

Comment:
suggestion: Add a top border using $color-border-inverse-weakest and $border-width-30